### PR TITLE
Text change from sources to integration

### PR DESCRIPTION
**Pull request by radekkaluzik**
https://issues.redhat.com/browse/RHCLOUD-30501

### CODE REVIEW

File: src/components/CloseModal.js [hunk 1]
```diff
@@ -53,8 +53,8 @@ CloseModal.propTypes = {
 };
 
 CloseModal.defaultProps = {
-  title: <FormattedMessage id="wizard.closeTitle" defaultMessage="Exit source creation?" />,
-  exitTitle: <FormattedMessage id="wizard.exitText" defaultMessage="Exit" />,
+  title: <FormattedMessage id="wizard.closeTitle" defaultMessage="Cancel creating the integration?" />,
```
Comment (fhlavac) [inline src/components/CloseModal.js:56]: @radekkaluzik maybe somebody from UX could review these bigger rewordings where are doing more than just switching Sources/Integrations. Maybe this is already present somewhere in the JIRA, but I haven't found it

File: src/components/CloseModal.js [hunk 2]
```diff
@@ -53,8 +53,8 @@ CloseModal.propTypes = {
...
+  exitTitle: <FormattedMessage id="wizard.exitText" defaultMessage="Cancel" />,
   stayTitle: <FormattedMessage id="wizard.stayText" defaultMessage="Stay" />,
```
Comment (fhlavac) [inline src/components/CloseModal.js:58]: Per yesterday's discussion about similar things in RBAC, these should probably be "Discard" and "Cancel"
https://redhat-internal.slack.com/archives/C05M0SNTLM8/p1706801711430029

File: src/components/SourceRemoveModal/SourceRemoveModal.js [hunk 3]
```diff
@@ -104,14 +104,14 @@ const SourceRemoveModal = ({ backPath = '/' }) => {
       className="sources"
       aria-label={intl.formatMessage({
         id: 'sources.deleteTitle',
-        defaultMessage: `Remove source?`,
+        defaultMessage: `Remove integration?`,
```
Comment (fhlavac) [inline src/components/SourceRemoveModal/SourceRemoveModal.js:107]:
```suggestion
        defaultMessage: 'Remove integration?',
```

File: src/components/SourceRemoveModal/SourceRemoveModal.js [hunk 4]
```diff
@@ -104,14 +104,14 @@ const SourceRemoveModal = ({ backPath = '/' }) => {
...
       })}
       header={
         <Title headingLevel="h1" size="2xl" className="sources">
           <ExclamationTriangleIcon size="sm" className="ins-m-alert src-c-delete-icon pf-v5-u-mr-sm" />
           {intl.formatMessage({
             id: 'sources.deleteTitle',
-            defaultMessage: `Remove source?`,
+            defaultMessage: `Remove integration?`,
```
Comment (fhlavac) [inline src/components/SourceRemoveModal/SourceRemoveModal.js:114]:
```suggestion
            defaultMessage: 'Remove integration?',
```

**Comment by fhlavac** [comment]
@radekkaluzik looks great! Just some nitpicking

File: src/components/CloseModal.js (hunk 1 above)
Comment (radekkaluzik) [reply src/components/CloseModal.js:56]: This change was done with the approval of the UX team.

File: src/components/CloseModal.js (hunk 2 above)
Comment (radekkaluzik) [reply src/components/CloseModal.js:58]: Same as previous comment. This was directly suggested by UX team.

File: src/components/SourceRemoveModal/SourceRemoveModal.js (hunk 3 above)
Comment (radekkaluzik) [reply src/components/SourceRemoveModal/SourceRemoveModal.js]: Fixed

File: src/components/SourceRemoveModal/SourceRemoveModal.js (hunk 4 above)
Comment (radekkaluzik) [reply src/components/SourceRemoveModal/SourceRemoveModal.js]: Fixed

**Pull request merged**